.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+	"name": "wasmCloud",
+	"image": "rust:bullseye",
+	"features": {
+		"devwasm.azurecr.io/dev-wasm/dev-wasm-feature/rust-wasi:0": {},
```

Comment:
Is this just adding the wasi toolchain? If so, it's already managed in `rust-toolchain.toml`, so it should not be necessary